tests/e2e/specs/Tasks.spec.js

```diff
@@ -118,25 +118,4 @@ describe('Tasks', () => {
         expect(xml).to.contain(emptyCallActivityXml.trim());
       });
   });
-
-  it('Shows secondary dropdown for task user assignment', function() {
-    if (!Cypress.env('inProcessmaker')) {
-      this.skip();
-    }
-
-    const taskPosition = { x: 200, y: 200 };
-    dragFromSourceToDest(nodeTypes.task, taskPosition);
-
-    getElementAtPosition(taskPosition).click();
-
-    cy.contains('Task Assignment').next().select('User');
-    cy.contains('Assigned User')
-      .should('exist')
-      .next().find('.multiselect').should('exist');
-
-    cy.contains('Task Assignment').next().select('Group');
-    cy.contains('Assigned Group')
-      .should('exist')
-      .next().find('.multiselect').should('exist');
-  });
```

Comment:
This test seems to rely completely on ProcessMaker core so I thought deleting it was the best option.